crates/tx5-connection/src/lib.rs

```diff
@@ -33,6 +33,26 @@ compile_error!("Must specify exactly 1 webrtc backend");
 #[cfg(feature = "backend-go-pion")]
 pub use tx5_go_pion::Tx5InitConfig;
 
+/// Grace time to allow a slow app to catch up before we close a
+/// connection to prevent our memory from filling up with backlogged
+/// message data.
+const SLOW_APP_TO: std::time::Duration = std::time::Duration::from_millis(99);
```

Comment:
Why 99 ms?

Reply:
Originally it was 19ms to keep it under 20ms for tokio, but this version is compatible with async code so it can be a bit longer for some more leeway.